RELEASES.md

```diff
@@ -1,3 +1,144 @@
+Version 1.63.0 (2022-08-11)
+==========================
+
+Language
+--------
+- [Remove migrate borrowck mode.][95565]
+- [Stabilize `$$` in Rust 1.63.0.][95860]
+- [Modify MIR building to drop repeat expressions with length zero.][95953]
+- [Remove label/lifetime shadowing warnings.][96296]
+- [Stabilize `explicit_generic_args_with_impl_trait`.][96868]
+- [make `cenum_impl_drop_cast` deny-by-default.][97652]
+- [Prevent unwinding when `-C panic=abort` is used regardless declared ABI.][96959]
+- [lub: don't bail out due to empty binders.][97867]
+
+Compiler
+--------
+- [Stabilize the `bundle` native library modifier,][95818] also removing the
+  deprecated `static-nobundle` linking kind.
+- [Add Apple WatchOS compile targets\*.][95243]
+- [Add Windows application manifest to rustc-main.][96737]
+
+\* Refer to Rust's [platform support page][platform-support-doc] for more
+   information on Rust's tiered platform support.
+
+Libraries
+---------
+- [Implement `Copy`, `Clone`, `PartialEq` and `Eq` for `core::fmt::Alignment`.][94530]
+- [Extend `ptr::null` and `null_mut` to all thin (including extern) types.][94954]
+- [`impl Read and Write for VecDeque<u8>`.][95632]
+- [STD support for the Nintendo 3DS.][95897]
+- [Make write/print macros eagerly drop temporaries.][96455]
+- [Implement `[OsStr]::join`.][96881]
+- [Implement `Hash` for `core::alloc::Layout`.][97034]
+- [os str capacity documentation.][97202]
+- [Put a bound on collection misbehavior.][97316]
+- [Stabilize checked slice->str conversion functions.][97367]
```

Comment:
```suggestion
```

This is already covered in the `These APIs are now usable in const contexts` section